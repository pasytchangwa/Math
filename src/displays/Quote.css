.quotation {
  width: 90%;
  font-size: 2rem;
  text-align: center;
  line-height: 3rem;
  height: 800px;
  margin: 0 auto;
  display: flex;
  align-items: center;
}