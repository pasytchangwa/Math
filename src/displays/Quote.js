import React from 'react';
import './Quote.css';

const Quote = () => (
    <div className="quotation">
      <i>
        Mathematics as an expression of the human mind reflects the active will,
        the contemplative reason, and the desire for aesthetic perfection. Its
        basic elements are logic and intuition, analysis and construction,
        generality and individuality.
      </i>
    </div>
  );

export default Quote;
