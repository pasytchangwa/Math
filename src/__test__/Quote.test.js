import renderer from 'react-test-renderer';
import React from 'react';
import Quote from '../displays/Quote';

test('Should render correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});

