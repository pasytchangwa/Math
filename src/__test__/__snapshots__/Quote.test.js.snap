// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div
  className="quotation"
>
  <i>
    Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality.
  </i>
</div>
`;
